phpBB/includes/acp/acp_forums.php

```diff
@@ -676,6 +678,7 @@ function main($id, $mode)
 					'S_ENABLE_INDEXING'			=> ($forum_data['enable_indexing']) ? true : false,
 					'S_TOPIC_ICONS'				=> ($forum_data['enable_icons']) ? true : false,
 					'S_DISPLAY_SUBFORUM_LIST'	=> ($forum_data['display_subforum_list']) ? true : false,
+					'S_DISPLAY_SUBFORUM_LIMIT'	=> ($forum_data['display_subforum_limit']) ? true : false,
 					'S_DISPLAY_ON_INDEX'		=> ($forum_data['display_on_index']) ? true : false,
 					'S_PRUNE_ENABLE'			=> ($forum_data['enable_prune']) ? true : false,
 					'S_PRUNE_SHADOW_ENABLE'			=> ($forum_data['enable_shadow_prune']) ? true : false,
```

Comment:
Not straightly related but probably a chance to change this kind of stuff to something like``` (bool) $forum_data['some_forum_data']``` 😛

Reply:
If I start here then that's going to a lot of refactoring. I do agree but also think this PR is not the place for that.